src/encoded/static/components/cart/actions.js

```diff
@@ -244,7 +244,7 @@ export const setCartStatus = status => (
  * @return {Promise} Resolves to the updated cart object
  */
 export const setCartNameIdentifierAndSave = ({ name, identifier }, cart, user, fetch) => (
-    (dispatch, getState) => (
+    dispatch => (
```

Comment:
Why did this change?

Reply:
getState isn’t used in this function, so it generated an ESlint error I hadn’t noticed before.